js/src/components/contact-information/contact-information-preview-card.scss

```diff
@@ -13,9 +13,9 @@
 	}
 
 	&__placeholder {
-		@include placeholder;
-
 		display: inline-block;
 		width: 18em;
+
+		@include placeholder;
```

Comment:
> Is this change necessary?

The change fixes a Sass deprecation warning. The explanation can be found in https://sass-lang.com/documentation/breaking-changes/mixed-decls/

![image](https://github.com/user-attachments/assets/49861298-ba96-4344-be87-0007085d1a03)

> I wonder if placeholder includes `display` or `width` props. Then it will be overrided

They don't have. Please refer to:

https://github.com/woocommerce/google-listings-and-ads/blob/fccc0cde1a9e21f1ddd9de9c5805895b7b080eae/js/src/css/abstracts/_mixins.scss#L9-L21